test/release/examples/primers/modules.chpl

```diff
@@ -0,0 +1,390 @@
+/*
+ * Module Primer
+ *
+ * This primer introduces the concept of modules, a strategy for encapsulating
+ * code for later use.  It covers:
+ *   how to define a module
+ *   namespace control for a module's symbols
+ *   outside access of a module's symbols
+ *   namespace control when using a module in an outside context, including:
```

Comment:
I'm tripping over "outside" again (primarily because I don't think of it as an adjective, and it seems non-technical).  Maybe drop "in an outside context"?  Or maybe even drop this entire bullet on the argument that it's subsumed by the "namespace control" bullet above?  (I realize that you're distinguishing between control within the module and outside the module, but is that distinction important in this overview?)